pilot/pkg/bootstrap/webhook.go

```diff
@@ -21,17 +21,14 @@ import (
 	"time"
 
 	"istio.io/pkg/log"
-
-	"istio.io/istio/pilot/pkg/features"
 )
 
 const (
 	HTTPSHandlerReadyPath = "/httpsReady"
 )
 
 func (s *Server) initHTTPSWebhookServer(args *PilotArgs) {
-	if features.IstiodService.Get() == "" {
-		log.Info("Not starting HTTPS webhook server: istiod address not set")
+	if s.kubeClient == nil {
```

Comment:
Are users still able to specify only run injector/validation on a remote cluster for single controlplane multicluster?

Reply:
Currently how will they run only injector/validation in other server? AFAICT, this change does not change that behaviour - but there may be subtle change that I am missing

Reply:
Actually the multi-cluster set up has been reported many bugs recently.  So i am not sure what's the status now.  
@linsun From https://github.com/istio/istio/issues/22934. I am not sure whether this (Use istiod to manage remote data planes) is the only choice in the future.

Reply:
Ok. Based on what is documented here https://github.com/istio/istio.io/issues/7243 and other issues, it seems this is not being used - I may be wrong here though, will wait for @linsun @howardjohn  to confirm

Reply:
There are two modes being worked on - run a minimal istiod in the remote, and run nothing in the remote. I don't think this PR changes either of those.

Reply:
@howardjohn For minimal istiod in the remote, i could not see why  `s.kubeClient ` should be initiated

Reply:
If it is not, then here it is not right.

Reply:
I think when minimal Istiod flag comes - we should address this. Do we have any other way to detect right now?

Reply:
As the single plane multicluster is down now. This is not a big problem atm.